internal/config/config.go

```diff
@@ -170,7 +173,7 @@ func (this *ConfigRecord) getFlags() error {
 	this.help = flag.Bool("h", false, "help")
 	this.noLexer = flag.Bool("no_lexer", false, "do not generate a lexer")
 	flag.StringVar(&this.outDir, "o", this.workingDir, "output dir.")
-	flag.StringVar(&this.pkg, "p", defaultPackage(this.outDir), "package")
+	flag.StringVar(&this.pkg, "p", "", "package")
```

Comment:
Is it possible to change this back to the defaultPackage function, since as far as I can remember, this prints when users type `--help`?

Reply:
Okay, fixed it.